packages/graphql/src/resolvers/IdentityEvents.js

```diff
@@ -160,170 +198,225 @@ function getAttestations(account, attestations) {
     })
     .filter(attestation => !!attestation)
 
-  return sortAttestations(verifiedAttestations)
+  return _sanitizeAttestations(verifiedAttestations)
 }
 
-function sortAttestations(attestations) {
-  const m = new Map()
-  // In case of a multiple events for same provider,
-  // only the latest one will be returned
-  attestations.forEach(att => m.set(att.id, att))
-
-  return getAttestationProviders().reduce((filtered, provider) => {
-    if (m.has(provider)) {
-      filtered.push(m.get(provider))
-    }
+/**
+ * Reads identity data associated with an Eth address from the identity server.
+ *
+ * @param {string} id: Eth address of the user.
+ * @returns {Promise<Object||null>} Returns the identity object or null if no identity found.
+ * @private
+ */
+async function _getIdentityFromBridgeServer(id) {
+  const identityServer = contracts.config.identityServer
+  if (!identityServer) {
+    throw new Error('identityServer server not configured')
+  }
+  const url = `${identityServer}/api/identity?ethAddress=${id}`
 
-    return filtered
-  }, [])
+  // Query the identity server.
+  const response = await fetch(url, { credentials: 'include' })
+  if (response.status === 204) {
+    // No identity found for this eth address.
```

Comment:
204?  Should this not be a 404?

Reply:
204 is No Content
404 is Not Found
I picked 204 because the route and the request are valid but there is just no identity data yet associated with the eth address. Rather than 404 which would imply the route is invalid.

Reply:
If we're not using status codes to refer to the URI(since it's a GET arg), then I'd argue it should still return 200.  But it's not worth holding this up or continuing to argue about it, so fine with me.